code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -860,6 +1033,105 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 		pref.bgstate = next_in_list(pref.bgstate, pref.bgstate_options)
 		return TOPIC_REFRESH_UPDATE_PREVIEW
 
+	else if(href_list["ear_style"])
+		// Construct the list of names allowed for this user.
+		var/list/pretty_ear_styles = list("Normal" = null)
+		for(var/path in ear_styles_list)
+			var/datum/sprite_accessory/ears/instance = ear_styles_list[path]
+			if(((!instance.ckeys_allowed) || (usr.ckey in instance.ckeys_allowed)) && (/*(!instance.apply_restrictions) || */(pref.species in instance.species_allowed)) || UnlockedToolbox)

```

Comment:
What's with the random commented out apply_restrictions?

Reply:
It's commented out as I've assumed you -want- the restrictions, otherwise you wouldn't specify them.

Reply:
... y'know what, looking at our own code, I have absolutely no idea what the point of even having that flag is. Fair enough x3